build.sbt

```diff
@@ -13,10 +14,8 @@ lazy val root = (project in file("."))
   .settings(inConfig(Test)(testSettings))
   .configs(IntegrationTest)
```

Comment:
IntegrationTest scope is now deprecated. Can we please follow these steps to use DefaultBuildSettings.itSettings() instead - https://github.com/hmrc/sbt-settings/blob/11dcc68b4003f0b0a6f3eaff2db63ca27d319030/README.md#integration-tests